md-dao/src/main/java/com/wipro/ats/bdre/md/dao/JobDAO.java

```diff
@@ -393,20 +388,18 @@
                 }
                 Criteria newTargetBatchCriteria = session.createCriteria(Batch.class).add(Restrictions.eq("instanceExec", instanceExec));
                 Batch newTargetBatch = (Batch) newTargetBatchCriteria.list().get(0);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "newTargetBatch" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1481)